etc/config/solidity.defaults.properties

```diff
@@ -34,10 +34,10 @@ group.resolc.intelAsm=false
 group.resolc.supportsBinaryObject=true
 group.resolc.supportsBinary=false
 group.resolc.supportsExecute=false
-compiler.resolc040_sol.exe=/opt/compiler-explorer/resolc-0.4.0/resolc
-compiler.resolc040_sol.semver=0.4.0
-compiler.resolc040_sol.name=resolc 0.4.0
-compiler.resolc040_sol.instructionSet=riscv64
+compiler.resolc050_sol.exe=/opt/compiler-explorer/resolc-0.5.0/resolc
+compiler.resolc050_sol.semver=0.5.0
+compiler.resolc050_sol.name=resolc 0.5.0
+compiler.resolc050_sol.instructionSet=riscv64
```

Comment:
Are these needed here? I don't see these keys in the other Solidity compiler default properties above.